features/accounts/edit/profile-form/profile-form.tsx

```diff
@@ -59,6 +60,15 @@ export const ProfileForm: FC = () => {
     // push(value);
   };
 
+  console.log(control);
+  const handleInputChange = event => {
+    const inputValue = event.target.value;
```

Comment:
This event handler looks like an attempt to provide manual validation of user input. This approach will not work. One possible workaround for implementing this function is to use the `setError` API from the `useForm` return in the `onChange` handler to select the date.
[Here's a rough guide](https://shahabyazdi.github.io/react-multi-date-picker/validation/#validating-input-value).